doc/topics/blocking-vs-non-blocking.md

```diff
@@ -0,0 +1,140 @@
+# Overview of Blocking vs Non-Blocking
+
+This overview covers the difference between blocking and non-blocking calls in
+Node.js. This overview will refer to the event loop and libuv but no prior
+knowledge of those topics is required. Readers are assumed to have a basic
+understanding of the JavaScript language and Node.js callback pattern.
+
+> In this overview, "I/O" refers primarily to interaction with the system's disk
+and network supported by [libuv](http://libuv.org/).
+
+
+## Blocking
+
+Blocking is when the execution of additional JavaScript in the Node.js process
+must wait until an I/O operation completes. Blocking may occur when using any of
```

Comment:
Would it be more appropriate to just take I/O out of the sentence or is there a better way to explain this? My goal is to distinguish that the process is waiting not *just* because JavaScript is executing.